bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/toolbar-submenu/component.jsx

```diff
@@ -68,8 +149,29 @@ export default class ToolbarSubmenu extends Component {
     }
   }
 
+  formatSubmenuLabel(type, obj) {
+    const { intl } = this.props;
+
+    if (type === 'annotations') {
+      const intlLabel = `tool${_.upperFirst(obj.value)}`;
+      return intl.formatMessage(intlMessages[intlLabel]);
+    }
+
+    if (type === 'color') {
+      const intlLabel = `color${_.upperFirst(obj.label)}`;
+      return intl.formatMessage(intlMessages[intlLabel]);
+    }
+
+    return (obj.label || obj.value).toString();
```

Comment:
I think `.toString()` wasn't the part that was missing. It is actually not needed.
You forgot about the thickness submenu, and handled only `annotations` and `colors` here.. that's why it returned pure numbers for the thickness and didn't create strings.

Plus you probably should add spaces while creating string, otherwise they will look like: `colorcyan`, `coloreletricLime`, `toolline` etc.